Assets/StreamingAssets/Saves/MainMenu.txt

```diff
@@ -0,0 +1 @@
+{"saveObjectList":[{"mapName":"Empty map (1)","isMain":true,"saveObjectList":[]}]}
```

Comment:
Did you mean to make this empty map save?

Reply:
no my bad